week1/homework.md

```diff
@@ -48,61 +48,57 @@ Now add a `.gitignore` file with these two lines:
 
 `.gitignore` will make sure we don't commit all the files in the `node_modules` folder, which can get huge.
 
-We are going to be using `nodemon` to run our application. `nodemon` will watch for file changes, and when a file is changed it will rerun our app. To install `nodemon` run
+The first step is to install [express](https://expressjs.com/) which is a javascript web framework.
 
-`npm install --save-dev nodemon`
+Install `express` with the following command
 
-The `--save-dev` will save the `nodemon` as a developer dependency
+    npm install -s express
```

Comment:
```suggestion
    `npm install -s express`
```

Reply:
I prefer the tabbed result to the back-ticks, as it makes the code formatting span the entire line (can be seen [here](https://github.com/HackYourFuture-CPH/node.js/blob/add-tests-demo/week1/homework.md#lets-setup-the-project)). But I also don't want to bike-shed about it, so if you prefer only the backticks, we can go with that.

Reply:
Ahh i did not realize it formatted it like code, no problem then :)